core/modules/user/user.theme.inc

```diff
@@ -56,7 +56,7 @@ function theme_user_signature($variables) {
  *
  * @param $variables
  *   An associative array containing:
- *   - account: The user object to format.
+ *   - account: Any object with 'name', 'uid', and 'picture' proprties.
```

Comment:
...perhaps add "such as a user object"?

Reply:
I would use _Any object with the 'name', 'uid', and 'picture' properties. _properties_ has been written _proprties_.

Reply:
I agree that it'd be nice to still mention user objects as an example, since they make up the majority use cases.

Reply:
Would * An object with the 'name', 'uid', and 'picture' properties, for example a user object.* be fine?

Reply:
@kiamlaluno Yea, though I'd prefer to remove the word 'the' in your example.

Reply:
Yes, since *properties* is plural, removing the article makes sense. (I still need to change from *Italian mind* when writing in English. 😅)